crates/utils/src/signing.rs

```diff
@@ -46,12 +46,13 @@ pub fn verify_signed_commit_boost_message<T: HashTreeRoot>(
     message: &mut T,
     signature: &BlsSignature,
     public_key: &BlsPublicKey,
-    slot_hint: Option<Slot>,
+    _slot_hint: Option<Slot>,
     root_hint: Option<Root>,
     context: &Context,
 ) -> Result<(), Error> {
-    let fork_version = slot_hint.map(|slot| context.fork_version_for(context.fork_for(slot)));
```

Comment:
Good catch. But I think this is the implemetation invented by ourselved. Helix original algorithm doesn't use slot.

Reply:
You can see https://github.com/lu-bann/helix/pull/15/files#diff-3b2d9daf9eed98dd27c340ab94524e841d07f6dbf9fc729e15875b448c412d71R34-R43